packages/KModal/KModal.vue

```diff
@@ -149,7 +149,7 @@ export default {
   bottom: 0;
   left: 0;
   right: 0;
-  background-color: var(--KModalBackdrop, var(--black-45, color(black-45)));
+  background-color: rgb(11, 23, 45, .6);
```

Comment:
Shouldn't this still accept `--KModalBackdrop` for customization (e.g. Portal)?

The variable still exists in the docs.

Reply:
Whoops yes, sloppy on my part, good catch!

Reply:
Fixed